src/arrayMethodIncludes.js

```diff
@@ -4,8 +4,29 @@
  * Implement method includes
  */
 function applyCustomIncludes() {
-  [].__proto__.includes2 = function(valueToFind, fromIndex) {
-    // write code here
+  [].__proto__.includes2 = function(valueToFind, fromIndex = 0) {
+    let index = fromIndex;
+
+    if (index < 0) {
+      index = this.length + index;
+    }
+
+    if (index < 0) {
```

Comment:
what is the difference between this conditions and the previous one?

Reply:
rewrote